rocks-lib/src/package/version.rs

```diff
@@ -226,9 +226,15 @@ pub enum PackageVersionReq {
 }
 
 impl PackageVersionReq {
+    /// Returns a `PackageVersionReq` that matches any version.
+    pub fn any() -> Self {
+        PackageVersionReq::SemVer(VersionReq::default())
```

Comment:
thought: when we add support for dev manifests, we might want to add a new `PackageVersionReq::Any` variant.
This is fine for now.

Reply:
That sounds reasonable. Initially I tried implementing a `Latest` variant, but that requires some impossible refactors (as you need to have access to a packagedb at all times to query the latest version if need be). `Any` sounds easier to implement.